apps/demos/Demos/CardView/SimpleArray/Vue/data.ts

```diff
@@ -0,0 +1,127 @@
+export type Customer = {
```

Comment:
Let's make interface here (like in other frameworks)
```suggestion
export interface Customer {
```